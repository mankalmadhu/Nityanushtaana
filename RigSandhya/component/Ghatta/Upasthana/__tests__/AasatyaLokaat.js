import React from 'react';
import ಆಸತ್ಯಲೋಕಾತ್ from '../AasatyaLokaat';
import Renderer from 'react-test-renderer';


jest.mock('../../../Vishaya/Vishaya',()=>'ವಿಷಯ');
jest.mock('../../../Vishaya/Viniyoga',()=>'ವಿನಿಯೋಗ');

test("ಆಸತ್ಯಲೋಕಾತ್", () => {
   expect(Renderer.create(
    <ಆಸತ್ಯಲೋಕಾತ್ />
  ).toJSON()).toMatchSnapshot();

});