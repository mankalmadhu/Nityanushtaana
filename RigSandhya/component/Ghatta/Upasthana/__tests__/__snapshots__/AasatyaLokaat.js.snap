// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ಆಸತ್ಯಲೋಕಾತ್ 1`] = `
<ವಿನಿಯೋಗ>
  ಆಸತ್ಯಲೋಕಾದಾಶೇಷಾತ್ ಆಲೋಕಾಲ್ಲೋಕಪರ್ವತಾತ್ । ಯೇ ಸ೦ತಿ ಬ್ರಾಹ್ಮಣಾದೇವಾಃ ತೇಭ್ಯೋನಿತ್ಯ೦ ನಮೋನಮಃ ॥
</ವಿನಿಯೋಗ>
`;
